api/db/setup_local_database.sh

```diff
@@ -41,7 +39,7 @@ trap finish EXIT
 cat drop_db.sql create_db.sql | envsubst > $CREATE_DB_FILE
 
 echo "Creating empty database..."
-mysql -u "$ROOT_DB_USER" $ROOT_PASSWORD_ARGS < ${CREATE_DB_FILE}
+mysql -u "$ROOT_DB_USER" -p${ROOT_PASSWORD} < ${CREATE_DB_FILE}
```

Comment:
Yes! Sorry to take so long to get back to this. 
Basically, at least my computer was having a problem I believe with the chained variables? Something about the way the quotes were setup I suspect was causing it to literally pass in '{ROOT_PASSWORD}' as the password. (I couldn't actually get it to echo exactly what was being passed in)
This fixed it on my end, but it could be that there is a better fix that involves messing with the quotes or variables of some sort.